simapp/v2/app_di.go

```diff
@@ -171,7 +171,15 @@ func NewSimApp[T transaction.Tx](
 
 	// RegisterUpgradeHandlers is used for registering any on-chain upgrades.
 	app.RegisterUpgradeHandlers()
-
+	/****  Store Metrics ****/
```

Comment:
nit, can we just move this snippet under line 168 instead? then both store stuff are close to each other.